renovate.json

```diff
@@ -2,6 +2,7 @@
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
     "github>elastic/renovate-config:only-chainguard",
+    "github>elastic/renovate-config:ci-agent-images",
```

Comment:
On top of the previous not the preset `:ci-agent-images` matches specific package names. I wonder if we have checked it and we believe the wanted packages are in there. @jamietanna 's suggestion will include any ci image, also not sure if we want all ci agent images to be matched.